docs/sysadmin-deploy-on-prem/linux-packages.txt

```diff
@@ -194,6 +194,50 @@ Master and Agent
    the right-hand side of the top navigation bar, and if you select the ``Cluster`` view in the
    left-hand navigation panel.
 
+.. _socket-activation:
+
+Socket Activation
+=================
+
+The master can be configured to use `systemd socket activation
+<https://0pointer.de/blog/projects/socket-activation.html>`__, allowing it to be started
+automatically on demand (i.e., when a client makes a network connection to the port) and restarted
+with reduced loss of connection state. To switch to socket activation, run the following commands:
+
+.. code::
+
+   sudo systemctl disable --now determined-master
+   sudo systemctl enable --now determined-master.socket
+
+When socket activation is in use, the port on which the master listens is configured differently;
+the port listed in the master config file is not used, since systemd manages the listening socket.
+Instead, run
+
+.. code::
+
+   sudo systemctl edit determined-master.socket
+
+which will open a text editor window. To change the listening port, insert the following text (with
+the port number substituted appropriately) into the editor and then exit the editor:
+
+.. code::
+
+   [Socket]
+   ListenStream=
```

Comment:
it's odd this line is required but I've tried to remove it and it broke stuff 🤷

Reply:
There can be multiple values, so taking this out causes systemd to configure two sockets. This line tells it to clear the list so that we only get the one from the next line.